style.css

```diff
@@ -1,65 +1,54 @@
 body {
-    padding: 0;
-    margin: 0;
-    font-family: Arial, Helvetica, sans-serif;
-    background-color: rgb(226, 226, 226);
-}
-
-header {
-    text-align: center;
-}
-
-div.todo {
-    padding: 1em;
-}
+  padding: 0;
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #e2e2e2; }
```

Comment:
nit: from a standards point of view this is a little odd, just throw closing bracket on the next line. It'll also benefit future PRs if you add a new property because it will only mark one line as changed instead of two.

Reply:
As you are probably aware by now, its Sass output :) Typically I wouldn't commit the Sass output, but included it to make it easier to run the project.

Great point about the diff. I usually follow the tip out of convention but its good to know a solid advantage exists for it.